.github/workflows/PR-build.yml

```diff
@@ -253,6 +253,27 @@ jobs:
           ARCH=amd64 DEST=build/packages/debian/amd64 tools/packaging/debian/create_deb.sh
           ARCH=arm64 DEST=build/packages/debian/arm64 tools/packaging/debian/create_deb.sh
 
+  test-control-stript:
```

Comment:
Using PR-build for now to test that this works.

Reply:
I decided to leave this permanently since these tests are very cheap to execute.